composer/trainer/trainer.py

```diff
@@ -946,8 +1046,10 @@ def __init__(
             model=model,
             callbacks=callbacks,
             grad_accum=grad_accum,
-            auto_grad_accum=auto_grad_accum,
             eval_batch_split=eval_batch_split,
+            train_device_microbatch_size=train_device_microbatch_size,
+            auto_microbatching=auto_microbatching,
+            using_device_microbatch_size=using_device_microbatch_size,
```

Comment:
do we need all three of these variables. wouldn't just `train_device_microbatch_size` tell you all you need to know?

Reply:
`train_device_microbatch_size` is converted to batch_size if auto is configured. `using_device_microbatch_size` will go away post deprecation